### ISSUE TITLE
GLIBC_2.27' not found in 0.8.4

### ISSUE THREAD

**Issue by sporto** [avh4/elm-format#709]
We updated the node package from 0.8.3 to 0.8.4
When trying to run elm-format we get:

```
/usr/src/app/node_modules/elm-format/unpacked_bin/elm-format: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by /usr/src/app/node_modules/elm-format/unpacked_bin/elm-format)
```

**Comment by sporto** [comment]
This is inside a Docker Node image

Ubuntu with Node 12.14 from here https://hub.docker.com/_/node

```
uname -a
Linux 63c18b4da8af 5.4.0-7642-generic #46~1598628707~20.04~040157c-Ubuntu SMP Fri Aug 28 18:02:16 UTC  x86_64 GNU/Linux
```

**Comment by avh4** [comment]
Thanks, there was a report of this on slack too..   Can you tell which version of glibc is available?  2.27 was apparently released in Feb 2018 which seems like it should be old enough to be supported most places.  But also I didn't do anything specific to set the glibc version either (and the elm-format linux binaries are built in a docker container).

I don't know that much about modern linux, so if anyone has advice here (esp w/r to haskell), let me know!

**Comment by avh4** [comment]
Hmm, not that this is necessarily a viable solution, but apparently there are buster variants of the `node` docker images.

`node:12.14` fails with the GLIBC_2.27 not found error:
```
docker run -it node:12.14 /bin/sh -c "npm i elm-format && node_modules/.bin/elm-format --help"
```

`node:12.14-buster` works:
```
docker run -it node:12.14-buster /bin/sh -c "npm i elm-format && node_modules/.bin/elm-format --help"
```

**Comment by sporto** [comment]
This Ubuntu image has `2.24` available

**Comment by avh4** [comment]
Adding to the notes:

apparently the default `node` docker images are based on `buildpack-deps:stretch` (Debian 9), but there's also a corresponding `-buster` (Debian 10) version of each.  And as sporto notes, apparently stretch has glibc-2.24.

**Comment by avh4** [comment]
Also note that I want to switch the linux binaries to be statically linked (see https://github.com/avh4/elm-format/issues/589) which I think would also solve this issue once that can be done.

**Comment by kraklin** [comment]
I have just tried it and using `-buster` node docker image has helped 👍 Thanks ;)
```
FROM: node:12.8.4-buster
```

**Comment by avh4** [comment]
Note for Travis-CI users: adding `dist: focal` to your `.travis.yml` appears to give the needed glibc version.

**Comment by avh4** [comment]
I'm testing the next release, and `docker run -it node:12.14 /bin/sh -c "npm i elm-format@0.8.5-rc1 && node_modules/.bin/elm-format --help"` appears to work, so this should be fixed once 0.8.5 gets released (which will happen as soon as #722 is confirmed to be working)

**Comment by avh4** [comment]
Should be fixed in 0.8.5
